README.md

```diff
@@ -26,6 +55,29 @@ make
 sudo make install
 ```
 
+And, lastly, LLVM 4.0:
+
+```
+cd ~
+wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
+apt-get install clang-4.0 lldb-4.0
```

Comment:
The line 
`apt-get install clang-4.0 lldb-4.0`
needs to read
`sudo apt-get install clang-4.0 lldb-4.0`

Reply:
Also, do we know minimum RAM, disk, and CPU for an acceptable node?